Kernel/ACPI/MultiProcessorParser.cpp

```diff
@@ -21,7 +21,7 @@ UNMAP_AFTER_INIT OwnPtr<MultiProcessorParser> MultiProcessorParser::autodetect()
     auto floating_pointer = find_floating_pointer();
     if (!floating_pointer.has_value())
         return {};
-    auto parser = adopt_own_if_nonnull(new MultiProcessorParser(floating_pointer.value()));
+    auto parser = adopt_own_if_nonnull(new (std::nothrow) MultiProcessorParser(floating_pointer.value()));
```

Comment:
Can we export this `std::nothrow` out of `std`? This feels awfully wordy

Reply:
It's a good idea.

While we are at it, we could also give it a more descriptive name like `fallible`. What do you think?

Reply:
Hmmm bikeshedding the name...I'm not sure...
cc @bgianfo, your input is appreciated :P